source/collections/encrypted-collection.txt

```diff
@@ -0,0 +1,58 @@
+.. _encrypted-collection:
+
+=============================================
+Create a Collection with Queryable Encryption
+=============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:v6.0:`Queryable Encryption </core/queryable-encryption/>` allows you to 
+encrypt a subset of fields in your collection.
+
+Procedure
+---------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Click the :guilabel:`Create Collection` button.
+
+      From the :guilabel:`Collections` screen, click the 
+      :guilabel:`Create Collection` button.
+
+   .. step:: Enter the collection name.
+
+   .. step:: Click the :guilabel:`Advanced Collection Options` dropdown.
+
+      Check the :guilabel:`Queryable Encryption` option.
+
+   .. step:: Specify an :guilabel:`Encrypted Field`. 
+
+      Specify which fields should be encrypted and whether they should 
+      be queryable.
+
+      The following fields are optional:
```

Comment:
“The following fields must be used if no existing keyId was specified for at least one encrypted field, and have no effect otherwise:”?

Reply:
Done.